CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# AxeTools/cachingTrait Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+- The 2.x branch will be added to support php ^8.0
+
+## [1.0.0]
+
+### Added
+
+- Initial Release 1.0.0 ([#1](https://github.com/AxeTools/CachingTrait/pull/1))
+- Support for php ^7.0
+- `CacheTrait::setCache()` static method ([#1](https://github.com/AxeTools/CachingTrait/pull/1))
```

Comment:
We should place the PR links in the bottom like we are doing in the `README.md` file.  This already is the wrong PR reference and would need to be changed 6 different times vs 1 reference